tsconfig.json

```diff
@@ -0,0 +1,41 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "allowJs": true,
+    "checkJs": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "outDir": "dist",
+    "rootDir": "./",
+    "noEmit": true,
+    "strict": true,
+    "noImplicitAny": false,
+    "strictNullChecks": false,
+    "strictFunctionTypes": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
```

Comment:
ah maybe this is why we have to do `return undefined`? If so, I'd like to consider switching this to `false`. Curious what others think.

Reply:
Reading the docs on this, I understand what it's trying to prevent, but I just grump at having to do `return undefined` when `return` does the same thing.

But if that's the only way that it works then I guess I can ungrump and get over it. @samthor @devnook wdyt?

Reply:
I'm less fussed but I do think `return undefined` is liable to create more confusion than it solves (since it's really not a normal convention). But is this GTS' default config?